discretisedfield/tests/test_field.py

```diff
@@ -2709,6 +2710,27 @@ def _init_random(p):
 
     assert f.allclose(f.rfftn().irfftn(shape=f.mesh.n))
 
+    # test 1d rfft
+    assert f.allclose(f.rfftn().irfftn(shape=f.mesh.n).real)
+    assert df.Field(mesh, nvdim=1).allclose(f.rfftn().irfftn(shape=f.mesh.n).imag)
+
+    assert f.allclose(f.rfftn().irfftn(shape=f.mesh.n))
```

Comment:
Why do we check first `real` and `imag` individually and afterwards the the full field?

Reply:
I think it could be us just over-checking or left over from debugging. Would you like me to remove it?